include/IECore/MeshAlgo.h

```diff
@@ -47,6 +47,23 @@ namespace IECore
 namespace MeshAlgo
 {
 
+class TexturePrimVarNames
+{
+	public:
+		TexturePrimVarNames( const std::string &positionPrimVarName, const std::string &uvSetName );
+
+		std::string positionPrimVarName;
+		std::string uvSetName;
+
+		std::string uName() const;
```

Comment:
I think I already explained how for Cortex 10 there won't be separate names for u and v, so I don't think enshrining the old names in a new API makes sense.